src/controls/Keyboard.ts

```diff
@@ -54,8 +54,9 @@ export function Keyboard() {
     { keys: ['Shift'], fn: (boost) => set((state) => ({ controls: { ...state.controls, boost } })) },
     { keys: ['r', 'R'], fn: reset, up: false },
     { keys: ['.'], fn: () => set((state) => ({ editor: !state.editor })), up: false },
-    { keys: ['i', 'I'], fn: () => set((state) => ({ help: !state.help, leaderboard: false })), up: false },
-    { keys: ['l', 'L'], fn: () => set((state) => ({ help: false, leaderboard: !state.leaderboard })), up: false },
+    { keys: ['i', 'I'], fn: () => set((state) => ({ help: !state.help, leaderboard: false, pickcolor: false })), up: false },
+    { keys: ['l', 'L'], fn: () => set((state) => ({ help: false, leaderboard: !state.leaderboard, pickcolor: false })), up: false },
+    { keys: ['p', 'P'], fn: () => set((state) => ({ help: false, pickcolor: !state.pickcolor, leaderboard: false })), up: false },
```

Comment:
Sorry to be annoying, but could you put the properties alphabetically?

```suggestion
    { keys: ['p', 'P'], fn: () => set((state) => ({ help: false, leaderboard: false, pickcolor: !state.pickcolor })), up: false },
```

Reply:
Not being annoying at all. That helps me to learn the codebase. Thanks for catching. 
Let me know if it's correct now.